### PR TITLE
Adding Relay Stake Pool to registry.json

### PR DESCRIPTION
**Pull request by MichaelVanStan**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+        "social": {                                                            
+            "website": "https://relaypool.com",
+            "discord": "Relay Stake Pool",
+            "reddit": "Relay_ADA_Pool",
+            "instagram": " ",
+            "facebook": " ",  
+            "youtube": " ",
+            "twitter": " ",
+        },
+        "telegram": "relaystakepool",
```
Comment (leo42) [inline registry.json:14]: Please use a personal telegram handle,(No groups or pages)

**Review by leo42 — CHANGES_REQUESTED** [review]

**Review by MichaelVanStan — COMMENTED** [review]
altered telegram handle from stakepool to personal

**Comment by MichaelVanStan** [comment]
@leo42 please see change above whereby updating telegram handle from pool's to my personal.

Please let me know if you'd like any further changes

**Review by Bez625 — CHANGES_REQUESTED** [review]
Please can you fix? It looks like the json validation failed

**Comment by MichaelVanStan** [comment]
@Bez625 & @leo42 please see change, I've fixed the JSON format issue (apologies for that).

**Pull request merged**